src/components/StudyLegend.jsx

```diff
@@ -1,38 +1,51 @@
-import React from 'react';
+import React, { Component } from 'react';
 import Menu from './Menu.jsx';
 import CategoricalDisplay from './CategoricalDisplay.jsx';
+import NotificationBadge from './NotificationBadge.jsx';
 import { connect } from '../store/Connect';
 import { IndicatorIcon } from './Icons.jsx';
 
-const StudyLegend = ({
```

Comment:
Pls be more careful when handling merge conflicts! You can't just go around undoing what other people have done and have them be cool about it. Check my comment on trello and look at https://github.com/binary-com/SmartCharts/pull/383